### PR TITLE
RELATED: RAIL-3191 fix catalog-export on tiger

### PR DESCRIPTION
**Pull request by no23reason**
* prevent TypeScript errors in generated files in case there are no measures in the workspace for instance
* update the guide shown when the TIGER_API_TOKEN is not set - we always need to guide the user to create a new token as retrieving the value of existing token in impossible
* handle invalid values of workspace-id more gracefully (on both tiger and bear)

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: tools/catalog-export/src/loaders/tiger/index.ts [hunk 1]
```diff
@@ -84,22 +84,25 @@ async function getTigerClient(hostname: string, usernameFromConfig: string | nul
...
-            await open(tokenUrl, { wait: false });
+            logInfo(
+                "1. You should now see your default browser open at either your Tiger installation's home page " +
+                    "or login page (if so, please log in).",
+            );
 
             logInfo(
-                `You should now see your default browser open at either ${tokenUrl} or at ` +
-                    `your Tiger installation's login page. Once you obtain the token, please set the ${TigerApiTokenVariable} environment variable and try again.`,
+                "2. Once on your Tiger installation's home page, please " +
```
Comment (kandl) [inline tools/catalog-export/src/loaders/tiger/index.ts:95]: "Once you are"?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3137/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4788/ : SUCCESS in 12m 49s

File: tools/catalog-export/src/loaders/tiger/index.ts (hunk 1 above)
Comment (no23reason) [reply tools/catalog-export/src/loaders/tiger/index.ts:95]: 👍

Comment (no23reason) [reply tools/catalog-export/src/loaders/tiger/index.ts]: Fixed

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1230

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/937/ : SUCCESS in 57s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1205/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4791/ : SUCCESS in 12m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/321/ : SUCCESS in 9m 46s (non-voting)

**Pull request merged**